flexmeasures/data/services/scheduling.py

```diff
@@ -29,7 +29,7 @@
 
 
 def create_scheduling_job(
-    sensor_id: int,
+    sensor: [int | Sensor],
```

Comment:
A pipeline in my PR #564 failed on this line. It should probably be `sensor: int | Sensor,`.

Reply:
Caught this myself already.